pkg/kb-summarizer/report/report.go

```diff
@@ -5,6 +5,8 @@ import (
 	"fmt"
 	"sort"
 
+	"os/exec"
+
```

Comment:
nit: extra blank lines are preventing imports from being sorted, same in summarizer.go, rest looks good

Reply:
Does it look ok now?

Reply:
yeah it looks good now